CMakeLists.txt

```
@@ -840,3 +840,14 @@ INSTALL(
     DESTINATION ${FREEGLUT_INSTALL_CMAKECONFIGDIR}
     COMPONENT Devel
 )
+
+# Install GLUT man pages on Unix-like systems
+IF(UNIX AND FREEGLUT_BUILD_MAN_PAGES)
```

```suggestion
IF(UNIX AND FREEGLUT_INSTALL_MAN_PAGES)
```

now thinking about it `FREEGLUT_INSTALL_MAN_PAGES` is probably more appropriate